pkg/rpc/request/txBuilder.go

```diff
@@ -107,12 +111,16 @@ func CreateFunctionInvocationScript(contract util.Uint160, method string, param
 	if param == nil {
 		emit.Opcodes(script.BinWriter, opcode.NEWARRAY0)
 	} else if slice, err := param.GetArray(); err == nil {
-		err = ExpandArrayIntoScript(script.BinWriter, slice)
-		if err != nil {
-			return nil, err
+		if len(slice) == 0 {
+			emit.Opcodes(script.BinWriter, opcode.NEWARRAY0)
```

Comment:
Why don't you do this in the `ExpandArrayIntoScript` itself? Or in another wrapper function if it doesn't emit PACK?